c11threads.h

```diff
@@ -23,6 +23,10 @@ Main project site: https://github.com/jtsiomb/c11threads
 
 #define ONCE_FLAG_INIT	PTHREAD_ONCE_INIT
 
+#ifdef __APPLE__
+#define PTHREAD_MUTEX_TIMED_NP PTHREAD_MUTEX_NORMAL
+#endif
```

Comment:
Makes sense.  I added _C11THREADS_NO_TIMED_MUTEX to the timespec_get conditional as well.  Not really sure if timespec_get is tied to the existence of the timed mutex.